src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,32 @@
 package mate.jdbc;
 
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        System.out.println("*************** list all manufacturers *******************");
+        manufacturerDao.getAll().forEach(System.out::println);
+        System.out.println("*************** create new manufacturers *******************");
+        Manufacturer testManufacturer = new Manufacturer(null, "testManufacturer", "testCountry");
```

Comment:
It is better to create another constructor without ID
```suggestion
        Manufacturer mercedes = new Manufacturer("Mercedes", "Germany");
```